content/handbook/engineering/data-engineering/database-excellence/_index.md

```
@@ -2,110 +2,111 @@
+## ヘルプのリクエスト
 
-### サポートエスカレーション
+緊急事態、サポートのエスカレーション、責任あるチームの特定を含むデータベース Issue のヘルプを得るための完全なガイドについては、[データベース Issue のヘルプを得る](/handbook/engineering/data-engineering/database-excellence/help/) を参照してください。
 
-TBD
+### インシデントエスカレーション
+
+データベースインシデントエスカレーションでは、オンコールルーティングのために [incident.io](https://app.incident.io/gitlab/on-call/schedules/01JXJ7MN4T14008GQKWYYNT6E8) を使用します。
+
+* **スコープ**: Incident Manager On Call、Engineer On Call、および Security チームによって発生した GitLab.com S1 および S2 本番インシデント。GitLab Dedicated のサポートはコンサルティング的です。セルフマネージドのサポートは裁量的で、ケースバイケースで評価されます。
+* **エスカレーション**: インシデント Slack チャンネルで `/inc escalate` を使用します。緊急でない Issue については、[トリアージローテーション](#triage-rotations) を使用するか、`#s_database_excellence` に投稿します。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)